includes/wc-attribute-functions.php

```diff
@@ -185,9 +185,9 @@ function wc_attribute_taxonomy_id_by_name( $name ) {
  */
 function wc_attribute_label( $name, $product = '' ) {
 	if ( taxonomy_is_product_attribute( $name ) ) {
-		$name       = wc_attribute_taxonomy_slug( $name );
+		$slug       = wc_attribute_taxonomy_slug( $name );
 		$all_labels = wc_get_attribute_taxonomy_labels();
-		$label      = isset( $all_labels[ $name ] ) ? $all_labels[ $name ] : $name;
+		$label      = isset( $all_labels[ $slug ] ) ? $all_labels[ $slug ] : $slug;
```

Comment:
This change will result in the same output as before, so not sure if it is fixing the issue you reported.
I'm fine with this change to improving the variable names, but as I said, doesn't sounds like that will fix any issue for you.

Reply:
@claudiosanches yes it will result in the same output and it will solve the issue for me because later in the filter I'll know if it's a label for a custom attribute or for taxonomy attribute (the name will contain "pa_" prefix )